<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>twip on Shell&#39;s Home</title>
    <link>//blog.shell909090.org/tags/twip/</link>
    <description>Recent content in twip on Shell&#39;s Home</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en-us</language>
    <copyright>CC-BY-SA4.0</copyright>
    <lastBuildDate>Wed, 03 Nov 2010 17:39:00 +0800</lastBuildDate>
    
	<atom:link href="//blog.shell909090.org/tags/twip/index.xml" rel="self" type="application/rss+xml" />
    
    
    <item>
      <title>twip在hawkhost上问题的解决</title>
      <link>//blog.shell909090.org/blog/archives/1604/</link>
      <pubDate>Wed, 03 Nov 2010 17:39:00 +0800</pubDate>
      
      <guid>//blog.shell909090.org/blog/archives/1604/</guid>
      <description>这两天twip的api不正常，跑上去看看，有个错误。
**Fatal error**: Cannot redeclare class OAuthException in **/home/shellcom/public\_html/apis/include/OAuth.php** on line **8**  这时候，找到include/OAuth.php，改成这样。
#class OAuthException extends Exception { # // pass #}  问题就暂时解决了。
这是因为主机上新装了什么库，这个库自己也定义了OAuthException（会定义这种异常的，估计是OAuth库）。所以，把这个自定义的异常移除，问题就暂时解决了。</description>
    </item>
    
  </channel>
</rss>